TOOLS/lua/autoload.lua

```diff
@@ -177,9 +236,14 @@ function scan_dir(path, current_file, dir_mode, separator, dir_depth, total_file
     table.filter(files, function (v)
         -- The current file could be a hidden file, ignoring it doesn't load other
         -- files from the current directory.
-        if (o.ignore_hidden and not (prefix .. v == current_file) and string.match(v, "^%.")) then
+        local current = prefix .. v == current_file
+        if (o.ignore_hidden and not current and string.match(v, "^%.")) then
             return false
         end
+        if (not current and is_ignored(v)) then
```

Comment:
style, not parentheses

Reply:
Thanks, I'll fix this. Should I add a new commit to the PR? Or can I rebase & force-push PR?

Reply:
rebase and force push is ok